### DISCUSSION TITLE
LuckySEO not working?

### DISCUSSION THREAD

**Discussion by arkraieski** [lucky-media/cloud#23] [Q&A] [answered]
Hi, I am not sure if this is a bug or if I'm just missing something (I am new to Statamic), but we seem to be unable to set titles and meta descriptions for our pages in the site I'm building. In `resources/views/partials/_seo.antlers.html`, I didn't understand where the `lucky_seo` variable came from, but the meta tags that use it aren't working. Am I doing something wrong? It seems like I can manage the SEO fields for the pages, but the templating might need to be reworked? Thanks.

**Comment by arkraieski** [comment, marked as answer]
It looks like this starter kit is just missing the `App\Tags\LuckySeo` class that's in your Landtamic starter kit: https://github.com/lucky-media/landtamic/blob/main/app/Tags/LuckySeo.php

I ended up grabbing the file from there, and it seems to do the trick. Based on the release notes, it seems like it might have just been an oversight that the LuckySeo class isn't in this starter kit too? I think I might be all set with this problem basically, but would appreciate some confirmation that I'm allowed to use that class in the site I'm building with Cloud :) 

Thanks!

**Reply by lokmanm** [reply]
Hey @arkraieski you are correct, looks like we forgot to include the class!

We will release a fix ASAP, thanks for reporting this!
